### PR TITLE
Add sample for command message pattern

### LINKED ISSUE #92: Create sample for Command Message Pattern
**Description:**
Create sample for [Command Message](https://www.enterpriseintegrationpatterns.com/patterns/messaging/CommandMessage.html).

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Purpose
$subject

Fixes #92

### CODE REVIEW

File: command-message/command-message.bal [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+final http:Client slackClient = check new ("http://api.slack.com.balmock.io");
+
+public function main() returns error? {
+    map<string> userGroupsParams = {
+        name: "Team Leads Group",
+        description: "This group is for the team leads internal communication.",
+        include_count: "true",
+        team_id: "T12345678"
+    };
+    _ = check slackClient->/api/usergroups\.create.post(userGroupsParams, mediaType = "x-www-form-urlencoded", targetType = UserGroupCreationResponse);
```
Comment (manuranga) [inline command-message/command-message.bal]: we need to use the returned value.

File: command-message/command-message.bal [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    string 'handle;
+    boolean is_external;
+    int date_create;
+    int date_update;
+    int date_delete;
+    string created_by;
+    string? updated_by;
+    string? deleted_by;
+    string user_count;
+};
```
Comment (ShammiL) [inline command-message/command-message.bal:18]: Do we need all these fields?

Comment (manuranga) [reply command-message/command-message.bal:18]: +1 to reduce fields.
why is it `Usergroup` not `UserGroup`?

Comment (LakshanWeerasinghe) [reply command-message/command-message.bal:18]: updated.

File: command-message/command-message.bal [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    UserGroup usergroup?;
+    string 'error?;
+};
+
+final http:Client slackClient = check new ("http://api.slack.com.balmock.io");
+
+service /api/v1 on new http:Listener(8080) {
+    isolated resource function post createUserGroup(UserGroupCreateRequest userGroup) 
+    returns UserGroupCreationResponse|error {
+        map<string> payload = check userGroup.ensureType();
```
Comment (manuranga) [inline command-message/command-message.bal]: this line is redundant

**Review by manuranga — APPROVED** [review]

**Pull request merged**